pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+authors = [
+    {name="Zeid Zabaneh", email="zeid@mozilla.com"},
+]
+classifiers = [
+    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+]
+description =  "Unified Lando Project"
+dependencies = [
+    "django==5.0.rc1",
+    "mozilla_django_oidc",
+    "psycopg2-binary"
+]
```

Comment:
> I think we should still be using a `requirements.txt` with pinning for a deployed application.

This does not preclude that. I am still ironing out some proposals for deployment, but for example we could publish `lando` to PyPI and then have a Dockerfile that installs pinned requirements, including `lando` with hashes, etc... so that it can be trusted and reproducible.

Reply:
Gotcha. :+1: